shell/platform/common/cpp/client_wrapper/standard_codec.cc

```diff
@@ -259,6 +266,9 @@ void StandardCodecSerializer::WriteVector(
     ByteBufferStreamWriter* stream) const {
   size_t count = vector.size();
   WriteSize(count, stream);
```

Comment:
Ditto

Reply:
Same, the reading code only works if we always write the size.